pushdeploy/apiv1.py

```diff
@@ -74,12 +106,62 @@ def login():
     access_token = create_access_token(identity=username, expires_delta=timedelta(seconds=90))
     return jsonify(access_token=access_token), 200
 
+@bp.route('/cronjob', methods=['POST'])
+@jwt_required
+def cronjob():
+    image_tag = request.args['image_tag']
+    image_name = request.args['image_name']
+    name = request.args['name']
+    namespace = request.args['namespace']
```

Comment:
Unless this does something special like ensure that the parameters are passed, these assignments feel noisy any unnecessary since you "consume them" just a few lines below.

Reply:
That style actually does force the request to pass those parameters.